app/portainer/components/datatables/registries-datatable/registriesDatatable.html

```diff
@@ -54,21 +54,25 @@
             >
               <td>
                 <span class="md-checkbox" ng-if="$ctrl.accessManagement">
-                  <input id="select_{{ $index }}" type="checkbox" ng-model="item.Checked" ng-click="$ctrl.selectItem(item, $event)" />
+                  <input id="select_{{ $index }}" type="checkbox" ng-model="item.Checked" ng-click="$ctrl.selectItem(item, $event)" ng-disabled="!$ctrl.allowSelection(item)" />
                   <label for="select_{{ $index }}"></label>
                 </span>
-                <a ui-sref="portainer.registries.registry({id: item.Id})" ng-if="$ctrl.accessManagement">{{ item.Name }}</a>
+                <a ui-sref="portainer.registries.registry({id: item.Id})" ng-if="$ctrl.accessManagement && item.Id">{{ item.Name }}</a>
+                <span ng-if="$ctrl.accessManagement && !item.Id">{{ item.Name }} (anonymous)</span>
```

Comment:
As `DockerHubViewModel` will have the name `Dockerhub (anonymous)` the `(anonymous)` addition is not needed here